dev/AnimatedVisualPlayer/AnimatedVisualPlayer.cpp

```diff
@@ -932,6 +981,11 @@ void AnimatedVisualPlayer::UpdateContent()
         // NOTE: If !IsAnimatedVisualLoaded() then this is a no-op.
         auto ignore = PlayAsync(from, to, looped);
     }
+
+    if (!Active())
+    {
+        Pause();
```

Comment:
Is this the correct place to check for Active state? right above we've called start, should it be in the check there?  Why do we even need to do this, if the AVP isn't active we don't ahve any animations so why do we need to pause?

Reply:
If AutoPlay is true we will start an animation on root.Progress property (a few lines above). Here I'm pausing this animation since there is nothing to animate yet. It will unpause when we set Active = true

Reply:
Changed it so that if Active==false, we will set AutoPlay to false (and will destroy any m_nowPlaying if it exists)